neo/Prompt/Commands/tests/test_wallet_commands.py

```diff
@@ -240,6 +257,61 @@ def test_wallet_create_address(self):
         self.assertEqual(type(res), UserWallet)
         self.assertEqual(len(res.Addresses), 9)
 
+    def test_wallet_claim_1(self):
+        # test with no wallet
+        args = ['claim']
+        res = CommandWallet().execute(args)
+        self.assertFalse(res)
+
+        self.OpenWallet1()
+
+        # test with max_coins_to_claim non integer
+        args = ['claim', 'abc']
+        claim_tx, relayed = CommandWallet().execute(args)
+        self.assertEqual(claim_tx, None)
+        self.assertFalse(relayed)
+
+        # test with max_coins_to_claim <= 0
+        args = ['claim', '-1']
+        claim_tx, relayed = CommandWallet().execute(args)
+        self.assertEqual(claim_tx, None)
+        self.assertFalse(relayed)
+
+        # test wrong password
+        with patch('neo.Prompt.Commands.Wallet.prompt', side_effect=["wrong"]):
+            args = ['claim']
+            claim_tx, relayed = CommandWallet().execute(args)
+            self.assertEqual(claim_tx, None)
+            self.assertFalse(relayed)
+
+        # test successfull
+        with patch('neo.Prompt.Commands.Wallet.prompt', side_effect=[WalletFixtureTestCase.wallet_1_pass()]):
+            args = ['claim']
+            claim_tx, relayed = CommandWallet().execute(args)
+            self.assertIsInstance(claim_tx, ClaimTransaction)
+            # self.assertTrue(relayed)
+
+        # test nothing to claim anymore
+        # with patch('neo.Prompt.Commands.Wallet.prompt', side_effect=[WalletFixtureTestCase.wallet_1_pass()]):
```

Comment:
I think that this test is failing because of the previous `relayed` to be false.